File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--inserting-data-into-a-workbook-on-a-server_2.vb
        Dim productDataSet As New AdventureWorksDataSet.AdventureWorksLTDataSet()
        Dim productTableAdapter As _
            New AdventureWorksDataSet.AdventureWorksLTDataSetTableAdapters.ProductTableAdapter()

        Dim workbookPath As String = System.Environment.GetFolderPath( _
            Environment.SpecialFolder.MyDocuments) & _
            "\AdventureWorksReport\bin\Debug\AdventureWorksReport.xlsx"
        Dim serverDocument1 As ServerDocument = Nothing

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--inserting-data-into-a-workbook-on-a-server_2.cs
            AdventureWorksDataSet.AdventureWorksLTDataSet productDataSet = 
                new AdventureWorksDataSet.AdventureWorksLTDataSet();
            AdventureWorksDataSet.AdventureWorksLTDataSetTableAdapters.ProductTableAdapter productTableAdapter =
                new AdventureWorksDataSet.AdventureWorksLTDataSetTableAdapters.ProductTableAdapter();

            string workbookPath = System.Environment.GetFolderPath(
                Environment.SpecialFolder.MyDocuments) +
                @"\AdventureWorksReport\bin\Debug\AdventureWorksReport.xlsx";
            ServerDocument serverDocument1 = null;